### DISCUSSION TITLE
postgres operator running but DB cluster POD initiation not happening  kubernetsv1.22.5

### DISCUSSION THREAD

**Discussion by vponnathota** [zalando/postgres-operator#2211] [General]
postgres operator running but DB cluster pod initiation not happening  in Kubernets v1.22.5 with ghcr.io/zalando/spilo-15:2.1-p9
Dynamic NFS storage available in my cluster with proper storage class is ready. I've followed CRD based config manifest files only not using configmap here. kubernets etcd only no external ETCD with 1 master and 3 workers are up and running fine. I'm not using any load balancer here right now i just want to run simple postgres DB setup with primary and standby include timescale extension and later i'll add LB,backups...etc. My major concern here DB pod initiation itself not happening.

**Comment by vponnathota** [comment]
[cmplete-manifest-postgres.txt](https://github.com/zalando/postgres-operator/files/10687690/cmplete-manifest-postgres.txt)

[operator-service-account-rbac.txt](https://github.com/zalando/postgres-operator/files/10687668/operator-service-account-rbac.txt)
[minimal-postgres-manifest.txt](https://github.com/zalando/postgres-operator/files/10687648/minimal-postgres-manifest.txt)
[operatorconfiguration.crd.txt](https://github.com/zalando/postgres-operator/files/10687649/operatorconfiguration.crd.txt)
[postgres-operator.txt](https://github.com/zalando/postgres-operator/files/10687651/postgres-operator.txt)
[postgresql-operator-default-configuration.txt](https://github.com/zalando/postgres-operator/files/10687652/postgresql-operator-default-configuration.txt)

![image](https://user-images.githubusercontent.com/75770729/217591565-1672b84e-4d04-45b9-ab11-c355b18f5ae3.png)

**Comment by dmvolod** [comment]
Hello @vponnathota 
Could you please provide or look at the operator pod logs
But, I can guess, that you are uncomment CONTROLLER_ID env in the operator deployment, appropriate `acid.zalan.do/controller` annotation should be present in the PostgreSQL CR.
